doc/nrf/known_issues.rst

```diff
@@ -1585,6 +1747,16 @@ nrfx_uart driver
 tx_buffer_length set incorrectly
   The nrfx_uart driver might incorrectly set the internal tx_buffer_length variable when high optimization level is set during compilation.
 
+Trusted Firmware-M (TF-M)
+*************************
+
+.. rst-class:: v1-8-0
+
+NCSDK-10022: Missing AES-CFB support in Nordic Security Module
+  AES-CFB is currently not supported in :ref:`nrfxlib:nrf_security` (nrf_security).
+  This leads to an issue trying to run TF-M regression tests.
+  The tests are disabled in the |NCS|.
```

Comment:
@frkv , can you check if the phrasing is OK?

Reply:
I would change the last line to `The tests using AES-CFB are disabled in the |NCS|`